authlib/common/cookie_manager.py

```diff
@@ -0,0 +1,6 @@
+import streamlit as st
+import extra_streamlit_components as stx
+
+@st.experimental_singleton
```

Comment:
Using experimental_memo does not work and
As far as I read the official documentation, experimental_singleton was more appropriate, so I used that one.